packages/core/useElementSize/index.ts

```diff
@@ -35,9 +35,10 @@ export function useElementSize(
           ? entry.contentBoxSize
           : entry.devicePixelContentBoxSize
 
-      if (boxSize) {
-        width.value = boxSize.reduce((acc, { inlineSize }) => acc + inlineSize, 0)
-        height.value = boxSize.reduce((acc, { blockSize }) => acc + blockSize, 0)
+      const formatBoxSize = Array.isArray(boxSize) ? boxSize : [boxSize]
+      if (formatBoxSize) {
```

Comment:
`boxSize` may be `undefined`, so formatBoxSize may goes `[ undefined ]` and go into the if block. 
Put the check inside if block would be better.

Reply:
Is a good suggestion

Reply:
I have fixed it, please review again @vaakian